io.github.go_musicfox.go-musicfox.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.github.go_musicfox.go-musicfox</id>
+  
+  <name>Musicfox</name>
+  <developer_name>anhoder</developer_name>
+  <summary>Terminal music player for Netease Cloud Music</summary>
+  <summary xml:lang="zh_CN">网易云音乐命令行客户端。</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <!--Not possible to control this application with mouse or touch since it only has a TUI-->
+  <recommends>
+    <control>keyboard</control>
+  </recommends>
+  
```

Comment:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#provides -> binary